kernel-api/src/main/scala/org/apache/toree/magic/MagicLoader.scala

```diff
@@ -29,7 +30,7 @@ class MagicLoader(
   urls: Array[URL] = Array(),
   parentLoader: ClassLoader = null
 ) extends URLClassLoader(urls, parentLoader) {
-  private val magicPackage = "com.ibm.spark.magic.builtin"
+  private val magicPackage = "org.apache.toree.magic.builtin"
```

Comment:
Ideally, we need a better system for detecting plugins (and support more kinds of plugins). I started working on this awhile ago using https://github.com/bmc/classutil to detect classes via annotations/interfaces. It uses the ASM bytecode library underneath, so the classutil library performs pretty well (and can work with jars that are not yet loaded).